build/commands/lib/checkNodeNpmVersion.js

```diff
@@ -0,0 +1,68 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+const assert = require('assert')
+const Log = require('./logging')
+
+// Define the required versions.
+const requiredNodeVersionRange = ['20.0.0', '*']
```

Comment:
FWIW, I think some of the most recent node versions don't work properly - I've had limited success with v22